src/app/components/media/ReportDesigner/ReportDesignerTopBar.js

```diff
@@ -32,6 +32,7 @@ const useStyles = makeStyles(theme => ({
   },
   reportHeader: {
     backgroundColor: 'var(--brandBackground)',
+    borderLeft: 'solid 2px var(--grayBorderMain)',
```

Comment:
This was added temporarily, to keep the item page looking the same until we get around to those layout changes